packages/x-handlebars/readme.md

```diff
@@ -25,33 +25,51 @@ If you are building an Express app using [n-ui][n-ui] you can register the helpe
 const express = require('@financial-times/n-ui');
 
 const app = express({
-	helpers: require('@financial-times/x-handlebars')
+	helpers: {
+		// existing helpers
+		...require('@financial-times/x-handlebars')
```

Comment:
object rest/spread is in node 10 but you've said it's node 6 compatible. i know this is just an example but baseline es6 syntax would be clearer i think